src/main/java/com/elytradev/concrete/block/ConcreteBlock.java

```diff
@@ -126,29 +126,17 @@ public Block setSoundType(SoundType sound) {
 	@Override
 	@SideOnly(Side.CLIENT)
 	public BlockRenderLayer getBlockLayer() {
-		if (this.concreteTranslucent) {
-			return BlockRenderLayer.TRANSLUCENT;
-		} else {
-			return super.getBlockLayer();
-		}
+		return this.concreteTranslucent ? BlockRenderLayer.TRANSLUCENT : super.getBlockLayer();
```

Comment:
I generally discourage ternary for clarity reasons, especially in a commit intended as a cleanup/clarity improvement. Sometimes more verbose is more readable.